traffic/traffic_tests/traffic_info_test.cpp

```diff
@@ -40,6 +40,8 @@ class TestMwmSet : public MwmSet
 };
 }  // namespace
 
+/// @todo Need TRAFFIC_DATA_BASE_URL for this test.
```

Comment:
Nobody cares about TODOs in the code. Please either file a ticket or ignore.

Reply:
We care.

Reply:
Well, I do regulalr checks for todo. And IDE show them also ..